app/elements/routing.html

```diff
@@ -95,6 +95,12 @@
 					page.createTeam();
 					return;
 				}
+				if (qs[i].indexOf('invitedby=') === 0) {
+					var invitedBy = getQueryParam(ctx, 'invitedby');
```

Comment:
this function already iterates the query params so you can remove even the `for` above and make it smth like

```javascript

    if (!ctx.querystring) {
        return;
    }
    if(getQueryParam(ctx, 'teamCreate') === 'true'){
        return page.createTeam();
    }
    var invitedBy = getQueryParam(ctx, 'invitedby');
    var teamName = getQueryParam(ctx, 'teamname');
    if(!!invitedBy && !!teamName){
        return page.acceptTeamInvitation(invitedBy, teamName);
    }
```